### DISCUSSION TITLE
Add dynamic bpm to the BeatmapAttributeText component

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#21511] [Ideas] [answered]
Title says it all for the most part. but here's a map where the issue really shows itself, the map ranges from 34 bpm to over 300, yet as can be seen the BPM value remains static

https://user-images.githubusercontent.com/74463310/205502621-35b70040-7f7b-47f2-9209-249e9d146669.mp4

**Comment by peppy** [comment]
This would be better suited to its own component. I don't think that one should be overloaded to have any kind of dynamic content.

**Comment by mk56-spn** [comment, marked as answer]
#21519
